### PR TITLE
HTML Boilerplate: removed code backticks from heading & Project: Recipes: changed heading to 3 backticks

### LINKED ISSUE #24338: Style Guide - Foundations/HTML Foundations: Headings and Section Placement
# Foundations/HTML Foundations

This issue is part of the Style Guide Community Project, a way of offering members of The Odin Project community a chance to contribute to our curriculum. Users who wish to take part and contribute to any Style Guide issues should have thoroughly read the [layout style guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md) and [The Odin Project contributing guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md).
  
## Description of Issue

This issue ensures that headings are properly formatted and that all sections are placed in the correct order.

Each applicable lesson for this issue should meet the following bulleted requirements:

- The main headings use the Markdown `###` syntax.
- Sub-headings that are not inline use the Markdown `####` syntax.
- Sub-headings that are inline use the Markdown `**sub-heading**` (i.e. bold) syntax.
- Both headings and sub-headings use Wikipedia style Title Case.
- There is a newline before and after headings and non-inline sub-headings.
- The order of existing sections follows the below format:
	- Introduction
	- Lesson Overview
	- Custom Sections (lessons) / Pre-Assignment Sections (projects)
	- Assignment
	- Practice (lessons) / Post-Assignment Sections (projects)
	- Knowledge Check
	- Additional Resources
- Additionally, there should be an empty newline at the end of the file.
 
## Acceptance Criteria

- [ ] The following lessons are updated according to the issue description above:
	- [ ] [Introduction to HTML and CSS](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/introduction-to-html-and-css)
	- [ ] [Elements and Tags](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/elements-and-tags)
	- [ ] [HTML Boilerplate](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/html-boilerplate)
            - Remove the code backticks from the "What is the `lang` attribute?" heading
	- [ ] [Working with Text](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/working-with-text)
	- [ ] [Lists](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/lists)
	- [ ] [Links and Images](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/links-and-images)
	- [ ] [Commit Messages](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/commit-messages)
	- [ ] [Project: Recipes](https://www.theodinproject.com/lessons/foundations-recipes)
            - The "Tips on when to commit" heading should be a level 3 `###` heading

### PR DESCRIPTION
**Pull request by itsaflamingo**
…ged the heading in Project:Recipes to 3 backticks

Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [ ] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [x] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

Closes #24338

**2. This PR:**

- Removed the code backticks from the "What is the lang attribute?" heading
- Changed "Tips on when to commit" heading to level 3 instead of 4 
- 
**3. Additional Information:**

### CODE REVIEW

File: foundations/html_css/html-foundations/html-boilerplate.md [hunk 1]
```diff
@@ -43,13 +43,12 @@ This becomes more important later on when we learn about manipulating HTML with
...
+```html
 <!DOCTYPE html>
-<html lang="en">
-</html>
-~~~
+<html lang="en"></html>
+```
 
-#### What is the `lang` attribute? 
+#### What is the lang attribute?
```
Comment (thatblindgeye) [inline foundations/html_css/html-foundations/html-boilerplate.md:51]:
```suggestion
#### What Is the Lang Attribute?
```

File: foundations/html_css/html-foundations/html-boilerplate.md [hunk 2]
```diff
@@ -78,70 +77,70 @@ There are many more elements that can go within the head of an HTML document. Ho
...
 
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <title>My First Webpage</title>
-  </head>
+    <head>
+        <meta charset="UTF-8" />
+        <title>My First Webpage</title>
+    </head>
```
Comment (thatblindgeye) [inline foundations/html_css/html-foundations/html-boilerplate.md:87]: Is there a particular reason for adding an additional 2 whitespace characters here?

File: foundations/html_css/html-foundations/html-boilerplate.md [hunk 3]
```diff
@@ -78,70 +77,70 @@ There are many more elements that can go within the head of an HTML document. Ho
...
-    <h1>Hello World!</h1>
-  </body>
+    <head>
+        <meta charset="UTF-8" />
+        <title>My First Webpage</title>
+    </head>
+
+    <body>
+        <h1>Hello World!</h1>
+    </body>
```
Comment (thatblindgeye) [inline foundations/html_css/html-foundations/html-boilerplate.md:136]: Similar to the above question

File: foundations/html_css/html-foundations/project-recipes.md [hunk 4]
```diff
@@ -19,11 +19,12 @@ If you do not know how to set up a repository, follow the instructions found in
...
 If you are having trouble:
 
-- All Git commands need to be run from inside your project's folder (did you forget to `cd` into the `odin-recipes` folder?).
-- Ensure you followed the steps [here on Step 2.3](/paths/foundations/courses/foundations/lessons/setting-up-git#step-2-configure-git-and-github) to clone from GitHub with SSH.
-- Refer to the [workflow](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/git-basics) in the Git Basics Lesson.
+-   All Git commands need to be run from inside your project's folder (did you forget to `cd` into the `odin-recipes` folder?).
+-   Ensure you followed the steps [here on Step 2.3](/paths/foundations/courses/foundations/lessons/setting-up-git#step-2-configure-git-and-github) to clone from GitHub with SSH.
+-   Refer to the [workflow](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/git-basics) in the Git Basics Lesson.
+
+### Tips on when to commit
```
Comment (thatblindgeye) [inline foundations/html_css/html-foundations/project-recipes.md:26]:
```suggestion
### Tips on When to Commit
```

File: foundations/html_css/html-foundations/project-recipes.md [hunk 5]
```diff
@@ -43,27 +44,27 @@ Okay, that's enough Git for the moment -- time to actually build stuff!
...
-3.  Under the description, add an "Ingredients" heading followed by an **unordered list** of the ingredients needed for the recipe.
+3. Under the description, add an "Ingredients" heading followed by an **unordered list** of the ingredients needed for the recipe.
 4. Finally, under the ingredients list, add a "Steps" heading followed by an **ordered list** of the steps needed for making the dish.
 
 #### Iteration 4: Add More Recipes
 
 1. Add two more recipes with identical page structures to the recipe page you've already created.
 2. Don't forget to link to the new recipes on the index page. Also, consider putting all the links in an unordered list so they aren't all on one line.
-</div>
+ </div>
```
Comment (thatblindgeye) [inline foundations/html_css/html-foundations/project-recipes.md:67]: This space isn't necessary

**Review by thatblindgeye — CHANGES_REQUESTED** [review]
Thanks for making this contribution! I had some comments/change requests below. Additionally, can you be sure to go through all the lessons mentioned in the issue and make sure all headings follow the Wikipedia style title case as mentioned in the layout style guide?

File: foundations/html_css/html-foundations/html-boilerplate.md (hunk 1 above)
Comment (itsaflamingo) [reply foundations/html_css/html-foundations/html-boilerplate.md:51]: Of course! My apologies, I didn't realize there was more to do:)

File: foundations/html_css/html-foundations/html-boilerplate.md (hunk 2 above)
Comment (itsaflamingo) [reply foundations/html_css/html-foundations/html-boilerplate.md:87]: I did not realize I did, my mistake. I did look over the other files, to make sure they adhered to the style guide, and I must have mis clicked at some point. Sorry:(

**Comment by itsaflamingo** [comment]
Pull request sent!

**Comment by wise-king-sullyman** [comment]
Closing this PR because its commit is included in #24403

**Pull request closed without merging**